appeals-service-api/src/__tests__/appeals.spec.js

```diff
@@ -0,0 +1,220 @@
+const request = require('supertest');
+const { MongoClient } = require('mongodb');
+const uuid = require('uuid');
+const mongodb = require('../db/db');
+const app = require('../app');
+const { appealDocument } = require('../models/appeal');
+
+jest.mock('../db/db');
+
+async function createAppeal() {
+  const appeal = appealDocument;
+  appeal.id = uuid.v4();
+  await mongodb
+    .get()
+    .db('appeals-service-api')
+    .collection('appeals')
+    .insertOne({ _id: appeal.id, uuid: appeal.id, appeal });
+  return appeal;
+}
+
+describe('Appeals API', () => {
+  let connection;
+  let db;
+
+  beforeAll(async () => {
+    connection = await MongoClient.connect(process.env.MONGO_URL, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    });
+    db = await connection.db();
+
+    mongodb.get.mockReturnValue(connection);
+  });
+
+  afterAll(async () => {
+    await connection.close();
+    await db.close();
+  });
+
+  test('POST /api/v1/appeals - It responds with a newly created appeal', async () => {
+    const appeal = appealDocument;
+    const response = await request(app).post('/api/v1/appeals').send({
+      // lpaCode: 'E60000281',
```

Comment:
Oh dear. I missed this unwanted comment. It does not affect behaviour so I will remove on next PR.